tests/unit/src/test/scala/tests/WorkspaceSymbolLspSuite.scala

```diff
@@ -13,6 +14,16 @@ import tests.MetalsTestEnrichments._
 
 class WorkspaceSymbolLspSuite extends BaseLspSuite("workspace-symbol") {
 
+  override protected def initializationOptions: Option[InitializationOptions] =
+    Some(
+      InitializationOptions.Default.copy(
```

Comment:
I wonder if we could make something line `TestDefault` in the `BaseLspSuite` ? So that we don't repeat the init options everywhere.